internal/automation-manager/api/src/main/resources/application.properties

```diff
@@ -0,0 +1,19 @@
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://localhost:3306/automation_manager?useSSL=false&requireSSL=false
+spring.datasource.username=springuser
+spring.datasource.password=ThePassword
+spring.http.multipart.maxFileSize=500MB
+spring.http.multipart.max-request-size=500MB
+spring.http.multipart.enabled=true
+#SMTP credentials
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+spring.mail.username=codeophrenia@gmail.com
+spring.mail.password=qoqlgzcnwoddinga
```

Comment:
Are these actual credentials? If so, these need to be removed and placeholders need to be used instead.

Reply:
Those are actual credentials and will remove it